README.md

```diff
@@ -1,39 +1,120 @@
 # PSI Cardinality
 Private Set Intersection Cardinality protocol based on ECDH and Bloom Filters.
 
+## Requirements
+
+There are requirements for the entire project which each language shares. There also could be requirements for each target language:
+
+### Global Requirements
+These are the common requirements across all target languages of this project.
+
+- [Bazel](https://bazel.build)
+
```

Comment:
What about compilers? Does compilers come with Bazel?

Reply:
I’m under the impression bazel detects the compilers available on your system. Ex msvc, gcc, or clang. 

Emscripten is added specifically here in this repo. So I’m sure it is possible to customize the desired compiler to use.

Reply:
Yeah, but some words about compatible compilers would be good. Although I'm not sure which compilers are needed for this project.

Reply:
Any of the three would work. Preferably clang.